.github/workflows/license.yml

```
@@ -11,22 +11,45 @@ jobs:
   update:
     runs-on: ubuntu-latest
 
-
     steps:
       # Checks-out repo
       - uses: actions/checkout@v2
+      
+      # Creat New Branch
+      - name: Create New Branch
+        uses: peterjgrainger/action-create-branch@v2.0.1
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          branch: 'license'
+          
```

Is this needed? 
```suggestion
      # Creat New Branch
      - name: Create New Branch
        uses: peterjgrainger/action-create-branch@v2.0.1
        env:
          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
        with:
          branch: 'license'
          
```

Why not just 
```
git checkout -b "new-branch-unqiuecode"
```
Seems easier and less work managing dependencies